compiler/tests/Tests/Codegen.hs

```diff
@@ -96,7 +96,7 @@ verifyCppCommCodegen args baseName =
 verifyCppGrpcCodegen :: [String] -> FilePath -> TestTree
 verifyCppGrpcCodegen args baseName =
     testGroup baseName $
-        map (verifyFile options baseName cppTypeMapping "") templates
+        map (verifyFile options baseName (cppExpandAliases (type_aliases_enabled options) cppTypeMapping) "") templates
```

Comment:
No test is currently exercising this in grpc or comm. I don't know if we need to.

Reply:
We also need a test that actually compiles and exercises the cpp changes.

Reply:
Will update existing examples that use aliases.